modules/ROOT/pages/bind-api-configure-service-mesh-CLI.adoc

```diff
@@ -65,4 +75,50 @@ The binding becomes activate within a few seconds and the API is successfully re
 +
 After you bind your API, the API in API Manager is marked as Active. You are now ready to apply policies to protect your APIs.
 +
-image::api-manager-status-for-service-mesh.png[85%, 85%]
\ No newline at end of file
+image::api-manager-status-for-service-mesh.png[85%, 85%]
+
+[[bind_using_CLI]]
+
+== Binding the Service Using Anypoint Service Mesh
```

Comment:
Change this heading to 
> Binding an API created using Anypoint Service Mesh

Reply:
Went with "Bind an API Created Using Anypoint Service Mesh". Done! Thanks!